admin_guide/sysctls.adoc

```diff
@@ -54,6 +54,13 @@ The following sysctls are known to be namespaced:
 - *_fs.mqueue.*_*
 - *_net.*_*
 
+[NOTE]
```

Comment:
Instead of a note, I'd say something like,

```
Additionally, some of the sysctls in the *_net.*_* group are known to be namespaced.

To confirm which sysctls are namespaced for your kernel, run the following command:

----
$ command with variables
----

In this command:
* <variable> is ____
* <variable> is ____
```